server/pixabay_utils.py

```diff
@@ -0,0 +1,52 @@
+"""
+References:
+	* https://towardsdatascience.com/beyond-tags-and-entering-the-semantic-search-era-on-images-with-openai-clip-1f7d629a9978
+"""
+
+from io import BytesIO
+from PIL import Image
+import logging
+import requests
+import time
+
+ORIGINAL_API = "https://pixabay.com/api/?key="
+API_KEY = ""  # Change it
```

Comment:
@deep-diver if you have some suggestions on how we should tackle this here, please let me know.

Reply:
For deploying to app engine, we could make the API key as a GitHub Secret and inject it as an environment variable,

Or we can Encrypt it using GPG, what do you think?

Reply:
GitHub secret sounds good to me. I will send it over to you so that you can add it. Works?

Reply:
sure!

Reply:
But how should we retrieve that from GitHub secrets? Will it be available via `os.environ`?